generic/servicelog.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Copyright: 2017 IBM
+# Authors: Rafael Camarda Silva Folco (rfolco@br.ibm.com) (Original author)
+#          Lucas Meneghel Rodrigues (lucasmr@br.ibm.com)
+#          Hariharan T.S. <hari@linux.vnet.ibm.com> (ported to avocado)
+#          Pavithra <pavrampu@linux.vnet.ibm.com> (Added test_servicelog)
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.service import ServiceManager
+from avocado.utils.software_manager import SoftwareManager
+
+
+class servicelog(Test):
+
+    is_fail = 0
+    url = "http://ausgsa.ibm.com/projects/p/plx/FVT/rtas-events"
```

Comment:
@Pavithra1602 if there is no other way than i think we can kept it on data dir , thanks for helping